examples/db_connector/1_basic.py

```diff
@@ -1,8 +1,11 @@
+# %% [markdown]
 """
 1. Basics
 =========
+
 """
 
+# %%
```

Comment:
Where is `pip install dff`? We will have to check all examples, I added a special test for that

Reply:
Added to all remaining examples